src/dashboards/Compact/index.tsx

```diff
@@ -119,7 +119,14 @@ const EnturDashboard = (): JSX.Element | null => {
         boardId ? getFromLocalStorage(boardId + '-tile-order') : undefined,
     )
 
-    const bikeRentalStations = useBikeRentalStations()
+    const bicycleStopPlaces = getStopPlaces(FormFactor.BICYCLE)
+    const carStopPlaces = getStopPlaces(FormFactor.CAR)
```

Comment:
Ikke noe vi trenger i det hele tatt in the forseeable future men kunne vært nice med query batching
(disclaimer: bare food for thought, tror ikke vi bør implementere det, ville nok vært premature optimization, men sometime down the road eller på et annet prosjekt hvor det blir sykt mange requests kanskje?) https://www.apollographql.com/blog/apollo-client/performance/batching-client-graphql-queries/

Reply:
Helt enig! Det så bra ut, så nice å ha i bakhodet til senere 💃